Don't define _ABIO32/_ABIN32/_ABI64 as these are ABI-specific preprocessors.
We let gcc handle it since it will define the correct one depending on the
abi it is generating code for.  Also, our multilib headers rely on only 
certain ones being defined.

--- sysdeps/mips/sgidefs.h
+++ sysdeps/mips/sgidefs.h
@@ -55,19 +55,8 @@
 /*
  * Subprogram calling convention
  */
-#ifndef _ABIO32
-# define _ABIO32		1
-#endif
 #define _MIPS_SIM_ABI32		_ABIO32
-
-#ifndef _ABIN32
-# define _ABIN32		2
-#endif
 #define _MIPS_SIM_NABI32	_ABIN32
-
-#ifndef _ABI64
-# define _ABI64			3
-#endif
 #define _MIPS_SIM_ABI64		_ABI64
 
 #endif /* sgidefs.h */
